google/cloud/grpc_options.cc

```diff
@@ -43,6 +43,16 @@ grpc::ChannelArguments MakeChannelArguments(Options const& opts) {
   if (!user_agent_prefix.empty()) {
     channel_arguments.SetUserAgentPrefix(absl::StrJoin(user_agent_prefix, " "));
   }
+
```

Comment:
can we also copy over the comments, so we remember why we are using these settings.
https://github.com/googleapis/google-cloud-cpp/blob/315c752b080fc87350bf174697be8ace783efb08/google/cloud/storage/internal/storage_stub_factory.cc#L49-L59

Reply:
Also, can you remove that code block from `storage/internal/storage_stub_factory.cc`, now that the logic is handled in here?